### PR TITLE
chore: bump Go to 1.19

### PR DESCRIPTION
**Pull request by knqyf263**
## Description
Bump Go

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [ ] I've added tests that prove my fix is effective or that my feature works.
- [ ] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: .golangci.yaml
Comment (knqyf263) [inline .golangci.yaml:30]: structcheck, varcheck and deadcode are deprecated. unused is recommended now.

**Comment by knqyf263** [comment]
@owenrumney @chen-keinan Could either of you review this PR? I just bumped the Go version, but gofmt now tries to format comments as well. Therefore, I needed some tweaks on the comments. Otherwise, golangci-lint complains about it.

**Review by josedonizetti — APPROVED** [review]

**Review by chen-keinan — APPROVED** [review]

**Comment by chen-keinan** [comment]
> @owenrumney @chen-keinan Could either of you review this PR? I just bumped the Go version, but gofmt now tries to format comments as well. Therefore, I needed some tweaks on the comments. Otherwise, golangci-lint complains about it.

LGTM 🚀

**Review by tonaim — APPROVED** [review]

**Comment by knqyf263** [comment]
Looks like it requires review from Owen or Liam as they are code owner on `pkg/flag`. LIam is on vacation, so I'll wait for @owenrumney.

**Review by owenrumney — APPROVED** [review]
LGTM :+1:

**Pull request merged**
